packages/remirror__extension-react-code-block-language-select/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@remirror/extension-react-code-block-language-select",
```

Comment:
I feel this package name is too long. Maybe just `@remirror/extension-react-language-select`? Up to you.